Polygon_mesh_processing/test/Polygon_mesh_processing/test_pmp_locate.cpp

```diff
@@ -307,14 +337,18 @@ void test_helpers(const G& g, CGAL::Random& rnd)
   Face_location loc = PMP::random_location_on_face<FT>(f, g, rnd);
   std::set<face_descriptor> s;
   PMP::internal::incident_faces(loc, g, std::inserter(s, s.begin()));
-  assert(PMP::is_on_face_border(loc, g) || s.size() == 1);
+  if (std::is_same<K, EPECK>()) {
+    assert(PMP::is_on_face_border(loc, g) || s.size() == 1);
+  }
 
   loc = PMP::random_location_on_halfedge<FT>(h, g, rnd);
   std::vector<face_descriptor> vec;
   PMP::internal::incident_faces(loc, g, std::back_inserter(vec));
-  assert(PMP::is_on_vertex(loc, source(h, g), g) ||
-         PMP::is_on_vertex(loc, target(h, g), g) ||
-         vec.size() == 2);
+  if (std::is_same<K, EPECK>()) {
+    assert(PMP::is_on_vertex(loc, source(h, g), g) ||
+      PMP::is_on_vertex(loc, target(h, g), g) ||
+      vec.size() == 2);
+  }
```

Comment:
Not required (either it's precise enough, or the combinatorial part, `s.size()`, will be correct)